docs/using.rst

```diff
@@ -534,8 +534,90 @@ Example usage for DNS-01 (Cloudflare API v4) (for example purposes only, do not
    fi
 
 
+.. _managing-certs:
 
- 
+Managing certificates
+=====================
+
+To view a list of the certificates Certbot knows about, run
+the ``certificates`` subcommand:
+
+``certbot certifices``
+
+This will return information in the following format::
+
+  Found the following certs:
+    Certificate Name: example.com
+      Domains: example.com, www.example.com
+      Expiry Date: 2017-02-19 19:53:00+00:00 (VALID: 30 days)
+      Certificate Path: /etc/letsencrypt/live/example.com/fullchain.pem
+      Private Key Path: /etc/letsencrypt/live/example.com/privkey.pem
+
+``Certificate Name`` gives the name Certbot knows the certificate by. Pass this name
+to the ``--cert-name`` flag to specify a particular certificate for the ``run``,
+``certonly``, ``certificates``, ``renew`` and ``delete`` commands::
+
+  certbot certonly --cert-name example.com
+
+The ``--cert-name`` flag can also be used to modify the domains a certificate contains,
+by specifying new domains using the ``-d/--domains`` flag. If certificate ``example.com``
+previously contained ``example.com`` and ``www.example.com``, it can be modified to only
+contain ``example.com`` by specifying only ``example.com`` with the ``-d/--domains`` flag::
+
+  certbot certonly --cert-name example.com -d example.com
+
+The same format can be used to expand the set of domains a certificate contains, or to
+replace that set entirely::
+
+  certbot certonly --cert-name example.com -d example.org,www.example.org
+
+If a certificate is requested with ``run`` or ``certonly`` with a name that does not already
+exist, the new certificate created will be assigned the name specified.
+
+If your account key has been compromised or you otherwise need to revoke a certificate,
+use the revoke command to do so. Note that the revoke command is passed the certificate path
+(ending in ``cert.pem``), not a certificate name or domain. Additionally, if a certificate
+is a test cert obtained via the ``--staging/--test-cert`` flag, that flag must be passed to the
+``revoke`` subcommand::
+
+  certbot revoke --cert-path /etc/letsencrypt/live/CERTNAME/cert.pem
+
+Once a certificate is revoked (or for other cert management tasks), all of a certificate's
+relevant files can be removed from the system with the ``delete`` subcommand::
+
+  certbot delete --cert-name example.com
+
+For advanced certificate management tasks, it is possible to manually modify the certificate's
+renewal configuration file, located at ``/etc/letsencrypt/renewal/CERTNAME``.
+
+.. warning:: Modifying any files in ``/etc/letsencrypt`` can make it so Certbot can no longer
+   properly manage its certificates, and we do not recommend doing so for most users.
```

Comment:
Perhaps add: "For most tasks, it is safest to limit yourself to pointing symlinks at the files there, or using `--renew-hook` to copy / make new files based upon those files, if your operational situation requires it (for instance, combining certs and keys in different way, or having copies of things with different specific permissions that are demanded by other programs)."